column.go

```diff
@@ -331,9 +331,19 @@ func (cl *columnLoader) open(file *File, path []string) (*Column, error) {
 			// each page of the column should iterate through the pages and read
 			// the page headers to determine which compression and encodings are
 			// applied.
-			for _, encoding := range c.chunks[0].MetaData.Encoding {
-				c.encoding = LookupEncoding(encoding)
-				break
+			switch encodings := c.chunks[0].MetaData.Encoding; len(encodings) {
+			case 1:
+				c.encoding = LookupEncoding(encodings[0])
+			default: // If RLE, Plain, and an additional encoding are present, the additional encoding is the given one
+				for _, encoding := range encodings {
+					if encoding != format.Plain && encoding != format.RLE {
+						c.encoding = LookupEncoding(encoding)
+						break
+					}
+				}
+				if c.encoding == nil {
+					c.encoding = LookupEncoding(format.Plain)
```

Comment:
Is it possible that we might be wrongly selecting the encoding here? For example if the list of encodings was `[RLE, RLE]` (not sure if it's valid Parquet, but we would be deducting `PLAIN` even tho it wasn't present).

Reply:
Based on the way the writer wraps encodings this is correct (since something like RLE, RLE can't happen) but your point makes sense that maybe a different library wrote that.

Reply:
Updated, to use the default behavior in the nominal case